[Master] Add admin services docs (#3477)
wso2/docs-is: en/docs/apis/call-admin-services.md

# Call Admin Services

!!! warning

>     Admin services feature is deprecated from IS 6.0.0 onwards.

Reviewer: ```suggestion
    Admin services are deprecated from IS 6.0.0 onwards. You can use REST-based APIs.
```

administration capabilities through the UI.

A service in WSO2 IS is defined by the following components:

- Service component: provides the actual service.
- UI component: provides the Web user interface to the service.
- Service stub: provides the interface to invoke the service generated from the service Web Services Description Language (WSDL).

There can be instances where you want to call back-end Web services directly. For example, in test automation, to minimize the overhead of having to change automation scripts whenever a UI change happens, developers prefer to call the underlying services in scripts. The topics below explain how to discover and invoke these services from your applications.

---

## Discover the admin services

By default, the WSDLs of admin services are hidden from consumers. Given below is how to discover them using the OSGi console.

1. Add the following configuration to the `<IS_HOME>/repository/conf/deployment.toml` file.

       ``` 
       [admin_service.wsdl] 
       enable= true
       ```

> 2. Go to `<IS_HOME>/bin/` folder and start the WSO2 product as follows:

Reviewer: ```suggestion
2. Go to the `<IS_HOME>/bin/` folder and start the WSO2 product as follows:
```

    wso2server.bat -DosgiConsole
    ```

3. Once the server starts, hit the enter/return key several times to get the OSGI shell in the console.

> 4. In the OSGi shell, type: `listAdminServices` to view the list of admin services of your product.

Reviewer: ```suggestion
4. In the OSGi shell, type `listAdminServices` to view the list of admin services of your product.
```
